order/client_test.go

```diff
@@ -59,7 +59,16 @@ func CreateTestProductAndSku(t *testing.T) *stripe.SKU {
 func TestOrder(t *testing.T) {
 	sku := CreateTestProductAndSku(t)
 
+	couponParams := &stripe.CouponParams{
+		Amount:   99,
+		Currency: currency.USD,
+		Duration: coupon.Once,
+	}
+
+	c, err := coupon.New(couponParams)
```

Comment:
Minor, but currently the Go tests make at least a half-hearted attempt to clean up after themselves when they run. Maybe add this to the end of the test case:

``` go
coupon.Del(c.ID)
```

You can see another example of this in the [order update test case here](https://github.com/stripe/stripe-go/blob/2aea243be0e339e6a5687c6fa0b9675339221139/order/client_test.go#L226).